linode/manager
# fix: [APL-354] - disable shared CPU whenever APL is enabled

## Description 📝
Shared CPU's are currently unstable with APL, therefore we want to disable the entire table whenever APL is enabled.  

## Changes  🔄
List any change relevant to the reviewer.
- added 'shared' to LinodeTypeClass
- added isAPLEnabled to isPlanPanelDisabled
- created new notice for APL 

## Target release date 🗓️
November the 11th

## Preview 📷
![image](https://github.com/user-attachments/assets/13a1cb81-dc3d-4a91-a61b-4d4f8a2ab606)


## How to test 🧪
- Make sure APL feature flag is enabled and you are enrolled into the beta 
- go to create Kubernetes cluster page
- Enable APL
- Observe that the entire shared CPU table is disabled
